common/flavorization/wc_title_prefix.txt

```diff
@@ -0,0 +1,18 @@
+﻿# Kingdom
+grand_principality_nightborne = {
+	type = title
+	special = holder
+	tier = kingdom
+	priority = 126
+	cultures = { nightborne highborne }
+    governments = { feudal_government }
+}
+
+principality_nightborne = {
+	type = title
+	special = holder
+	tier = duchy
+	priority = 126
+	cultures = { nightborne highborne }
+    governments = { feudal_government }
+}
```

Comment:
I have a few more suggestions:
```suggestion
﻿# Kingdom
grand_principality_nightborne = {
	type = title
	tier = kingdom
	priority = 146 # Should be higher prior than duchy
	cultures = { nightborne highborne }
    governments = { feudal_government }
}

principality_nightborne = {
	type = title
	tier = duchy
	priority = 126
	cultures = { nightborne highborne }
    governments = { feudal_government }
}
```
And move it to `wc_title_holders.txt` just like it's in the vanilla.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114285420-0e2c6980-9a68-11eb-8c4a-3ef5071b7d24.png)

</details>

Also, I feel like we need to change the priorities of our cultural titles to be equal to the vanilla:
Baron - 7
Count - 18
Duke - 27
King - 47
Emperor - 102
Otherwise, we may end having some cultural title forcing kings to be called emperors since our cultural kings have more priority than default emperors.